### ISSUE TITLE
Background color doesn't change in SceneView

### ISSUE THREAD

**Issue by Wrakor** [ThomasGorisse/sceneform-android-sdk#14]
Hello,

I'm trying to use a SceneView to display a 3d object and the background color won't change:

<com.google.ar.sceneform.SceneView
            android:id="@+id/sceneView"
            android:background="@color/red_900"
            android:layout_width="match_parent"
            android:layout_height="match_parent" />

![image](https://user-images.githubusercontent.com/2973191/111509530-b8a0bd80-8744-11eb-9d25-e082865980e7.png)

**Comment by ThomasGorisse** [comment]
Try the following [https://github.com/google-ar/sceneform-android-sdk/issues/251#issuecomment-520056488](https://github.com/google-ar/sceneform-android-sdk/issues/251#issuecomment-520056488) until the new version is out.

**Comment by Wrakor** [comment]
> Try the following [google-ar#251 (comment)](https://github.com/google-ar/sceneform-android-sdk/issues/251#issuecomment-520056488) until the new version is out.

Didn't work, the background is still black.

**Comment by filanjuraj** [comment]
The only working solution I found was to add node with ViewRenderable:
```
private fun loadBackground() {
        ViewRenderable.builder()
            .setView(context, R.layout.scene_background)
            .build()
            .thenAccept { view ->
                 addBackgroundToScene(view)
            }    
}

private fun addBackgroundToScene(view: ViewRenderable?) {
        val bgNode = Node().apply {
            localScale = Vector3(4f, 4f, 4f)
            name = "bg"
            renderable = view
            localPosition = Vector3(0f, -2f, 0f)
        }

        sceneView.scene.addChild(bgNode)
}
```
Where scene_background is just some basic layout with fixed width and height, and with custom background (color or drawable).

**Comment by ThomasGorisse** [comment]
Fixed here: b6c6cee9d337922464e82c1b4e3819b85dd3462f
Please review the [SceneView Sample app](https://github.com/ThomasGorisse/sceneform-android-sdk/blob/38bcb379b569ebe079e8611c5bf1ef78749bcffe/samples/sceneview/src/main/java/com/google/ar/sceneform/samples/sceneview/MainActivity.java) for usage
Use the snapshot version as describe [here ](https://thomasgorisse.github.io/sceneform-android-sdk/dependencies)until next release.
